### PR TITLE
Projections

### PR DESCRIPTION
**Pull request by scsides**
Initial projection capabilities

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] I have read and agree to abide by the [Code of Conduct](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/Code-Of-Conduct.md)
- [X] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] I have added myself to the [.zenodo.json](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/.zenodo.json) document.
- [X] I have added any user impacting changes to the [CHANGELOG.md](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CHANGELOG.md) document.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [X] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

File: isis/appdata/import/PDS3/img_base.tpl [hunk 1]
```diff
@@ -82,10 +82,22 @@ Object = IsisCube
   End_Object
 
   Group = Instrument
+   i = 5
     {% block instrument %}
     {% endblock %}
+    i=6
   End_Group
+  i=7
```
Comment (Kelvinrr) [inline isis/appdata/import/PDS3/img_base.tpl]: I imagine these can be removed.

File: isis/appdata/import/PDS3/img_base.tpl [hunk 2]
```diff
@@ -82,10 +82,22 @@ Object = IsisCube
...
 
+  {% block mapping %}
+    {% if exists("IMAGE_MAP_PROJECTION") %}
+      Group = Mapping
+        {% if IMAGE_MAP_PROJECTION.MAP_PROJECTION_TYPE.Value == "EQUIRECTANGULAR" %}
+          {% include "Equirectangular.tpl" %}
+        {% endif %}
```
Comment (Kelvinrr) [inline isis/appdata/import/PDS3/img_base.tpl:94]: Seems like a reasonable way to reuse projection templates

**Review by Kelvinrr — COMMENTED** [review]
Looks like a decent enough solution for me so far.

File: isis/appdata/import/PDS3/Equirectangular.tpl [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+MinimumLatitude    = {{ IMAGE_MAP_PROJECTION.MINIMUM_LATITUDE.Value }}
+MaximumLatitude    = {{ IMAGE_MAP_PROJECTION.MAXIMUM_LATITUDE.Value }}
+MinimumLongitude   = {{ IMAGE_MAP_PROJECTION.EASTERNMOST_LONGITUDE.Value }}
+MaximumLongitude   = {{ IMAGE_MAP_PROJECTION.WESTERNMOST_LONGITUDE.Value }}
+UpperLeftCornerX   = -110.0
+UpperLeftCornerY   = -2366138.0
+PixelResolution    = {{ IMAGE_MAP_PROJECTION.MAP_SCALE.Value }} <meters/pixel>
+Scale              = {{ IMAGE_MAP_PROJECTION.MAP_RESOLUTION.Value }} <pixels/degree>
+TrueScaleLatitude  = {{ IMAGE_MAP_PROJECTION.CENTER_LONGITUDE.Value }}
+CenterLatitude     = {{ IMAGE_MAP_PROJECTION.CENTER_LATITUDE.Value }}
```
Comment (Kelvinrr) [inline isis/appdata/import/PDS3/Equirectangular.tpl:18]: might be worth moving these into their own folders.

**Comment by acpaquette** [comment]
This is still active

**Pull request closed without merging**